### PR TITLE
Move rawpacket to service

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: src/org/jitsi/service/neomedia/RawPacket.java [hunk 1]
```diff
@@ -13,10 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jitsi.impl.neomedia;
+package org.jitsi.service.neomedia;
 
 
-import org.jitsi.service.neomedia.*;
+import org.jitsi.impl.neomedia.*;
```
Comment (ibauersachs) [inline src/org/jitsi/service/neomedia/RawPacket.java]: Stuff from .service. shouldn't reference anything inside .impl. to truly follow the service vs. impl packaging

Comment (damencho) [inline src/org/jitsi/service/neomedia/RawPacket.java]: Ups. Checking it now.

Comment (gpolitis) [inline src/org/jitsi/service/neomedia/RawPacket.java]: @damencho would it make sense to use `org.jitsi.service...ByteArrayBuffer`? or does that break stuff?

**Comment by bgrozev** [comment]
LGTM

**Pull request merged**